src/components/divider/index.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Divider from ".";
+
+<Meta title="Components/Divider" component={Divider} />
+
+# Divider
+
+--- Add a description of the component here ---
+
+## Usage
+
+```jsx
+import { Divider } from "@wpmedia/arc-themes-components";
+
+const Feature = () => <Divider />;
+```
+
+## Properties
+
+<ArgsTable of={Divider} />
+
+## Stories
+
+** Divider **
+
+<Canvas>
+	<Story name="Divider">
+		<Divider />
```

Comment:
can you add a story with accessibility hidden as well? This would help show the capability of the storybook

Reply:
Thanks @JackHowa , fixed it in [00d9bb7](https://github.com/WPMedia/arc-themes-components/pull/145/commits/00d9bb796252649489f0047803e4e92507df9fb8)

Reply:
great thanks